src/app/pages/charging-stations/charging-limit/static-limit/charging-station-static-limit.component.html

```diff
@@ -29,7 +29,7 @@ <h3 class="card-title text-left">
           </div>
           <div class="col-md-12 text-right">
             <button mat-raised-button color="primary" (click)="applyStaticLimitConfirm(chargePoint)"
-              [disabled]="chargingStation.inactive || !chargePoint.ampCurrentLimit || (isSmartChargingComponentActive && chargingStation.siteArea.smartCharging)">
+              [disabled]="chargingStation.inactive || !chargePoint.ocppParamForPowerLimitation || !chargePoint.ampCurrentLimit || (isSmartChargingComponentActive && chargingStation.siteArea.smartCharging)">
```

Comment:
We have also supportStaticLimitation in charging station's capabilities.
This has to be also double checked in the backend.

Reply:
Ok, I will look at it to implement that properly.